src/main/java/org/folio/circulation/domain/ConfigurationService.java

```diff
@@ -76,7 +76,15 @@ private boolean isConfigurationEmptyOrUnavailable(JsonObject configJson) {
   }
 
   private Integer findTimeoutDuration(JsonObject configJson) {
-    return configJson.getInteger(CHECKOUT_TIMEOUT_DURATION_KEY, DEFAULT_CHECKOUT_TIMEOUT_DURATION_IN_MINUTES);
+    try {
+      return Integer.parseInt(configJson
+        .getValue(CHECKOUT_TIMEOUT_DURATION_KEY, DEFAULT_CHECKOUT_TIMEOUT_DURATION_IN_MINUTES).toString());
+    } catch (NumberFormatException e) {
+      log.debug("Can't parse property {} {}. Will be returned default value: {}",
+        CHECKOUT_TIMEOUT_DURATION_KEY, e.getMessage(),
+        DEFAULT_CHECKOUT_TIMEOUT_DURATION_IN_MINUTES);
```

Comment:
It is likely that this message will go unnoticed for some time. Is the default acceptable during that period?

Reply:
Agree, I'd use warn here or do propagate of exception.

Reply:
Replaced to warn